templates/books-index.html

```diff
@@ -0,0 +1,94 @@
+
+<div class="col-md-10 col-md-offset-1">
+  <form class="form-horizontal" ng-submit="bic.createBook()">
+    <fieldset>
+
+      <!-- Form Name -->
+      <legend>Add New Book</legend>
+
+      <!-- Text input -->
+      <div class="form-group">
+          <label class="col-md-4 control-label">Book Title</label>
+          <div class="col-md-4">
+            <input id="title" ng-model="bic.newBook.title" type="text" class="form-control input-md" required="">
+          </div>
+      </div>
+
+      <!-- Text input -->
+      <div class="form-group">
+          <label class="col-md-4 control-label">Book Author</label>
+          <div class="col-md-4">
+            <input id="author" ng-model="bic.newBook.author" type="text" class="form-control input-md" required="">
+          </div>
+      </div>
+
+      <!-- Text input -->
+      <div class="form-group">
+          <label class="col-md-4 control-label">Release Date</label>
+          <div class="col-md-4">
+            <input id="releaseDate" ng-model="bic.newBook.releaseDate" type="text" class="form-control input-md" required="">
+          </div>
+      </div>
+
+      <!-- Text input -->
+      <div class="form-group">
+          <label class="col-md-4 control-label">Image</label>
+          <div class="col-md-4">
+            <input id="image" ng-model="bic.newBook.image" type="text" class="form-control input-md" required="">
+          </div>
+      </div>
+
+      <!-- Button -->
+      <div class="form-group">
+        <label class="col-md-4 control-label">Save Book</label>
+        <div class="col-md-4">
+          <button ng-click="bic.createBook(bic.book)"class="col-md-4 btn btn-primary">Submit</button>
+        </div>
+      </div>
+
+      <!-- Search -->
+      <div class="form-group">
+      <label class="col-md-4 control-label">Search for a Book</label>
+      <div class="col-md-4">
+        <input type="text" class="form-control center-block col-md-4" ng-model="searchText" placeholder="Search Albums">
+      </div>
+
+    </fieldset>
+  </form>
+</div>
+
+
+<div ng-repeat="book in bic.books | orderBy: 'name' | filter: searchText" class="row">
+    <div class="col-md-10 col-md-offset-1">
+      <div class="panel panel-default">
+        <div class="panel-body">
+          <div class='row'>
+            <div class="col-md-3 col-xs-12 thumbnail">
+              <img class="book-image" src="{{ book.image }}" alt="book image">
+            </div>
+            <div class="col-md-9 col-xs-12">
+              <ul class="list-group form-inline">
+                <li class="list-group-item">
+                  <h4 class='inline-header'>Book title:</h4>
+                  <span> <a ng-href="books/{{book._id}}">{{ book.title }}</a></span>
+                </li>
+                <li class="list-group-item">
+                  <h4 class='inline-header'>Author: </h4>
+                  <span>{{ book.author }}</span>
+
```

Comment:
added whitespace line, not sure if this was intentional in the middle of a list item like this